Update stream-analytics-compatibility-level.md (#60435)
Azure/azure-content: articles/stream-analytics/stream-analytics-compatibility-level.md

**1.1 level:** CREATE TABLE allows you to specify a strong schema. The Stream Analytics engine validates that the data conforms to this schema. With this model, the command can filter events with NaN values.

### Disable automatic conversion of datetime strings to DateTime type at ingress for JSON

> **1.0 level:** The JSON parser would automatically convert string values with date/time/zone information to DATETIME type at ingress so the value will immediately lose original formatting and timezone information. Since this was done at ingress, even if that field was not used in the query it will now be converted into UTC DateTime.

Reviewer: ```suggestion
**1.0 level:** The JSON parser would automatically convert string values with date/time/zone information to DATETIME type at ingress so the value immediately loses its original formatting and timezone information. Since this is done at ingress, even if that field was not used in the query, it is converted into UTC DateTime.
```

> **1.1 level:** There is no automatic conversion of string values with date/time/zone information to DATETIME type. As a result, timezone information and original formatting are kept. However if the NVARCHAR(MAX) field is used in the query as part of a DATETIME expression (DATEADD function, for example) then it will be converted to DATETIME type to perform thecomputation and it loses its original form.

Reviewer: ```suggestion
**1.1 level:** There is no automatic conversion of string values with date/time/zone information to DATETIME type. As a result, timezone information and original formatting are kept. However if the NVARCHAR(MAX) field is used in the query as part of a DATETIME expression (DATEADD function, for example) then it will be converted to DATETIME type to perform the computation and it loses its original form.
```

> **1.1 level:** There is no more automatic conversion of string values with date/time/zone information to DATETIME type. As a result, timezone information and original formatting are kept. However if this NVARCHAR(MAX) field is used in the query as part of a DATETIME expression (DATEADD function for example) the it will still be converted to DATETIME type to perform computation and will lose original form.

Reviewer: ```suggestion
**1.1 level:** There is no automatic conversion of string values with date/time/zone information to DATETIME type. As a result, timezone information and original formatting are kept. However if the NVARCHAR(MAX) field is used in the query as part of a DATETIME expression (DATEADD function, for example) then it will be converted to DATETIME type to perform thecomputation and it loses its original form.
```
